### PR TITLE
COMP: Disable Elastix tests by default

### PR DESCRIPTION
**Pull request by tbirdso**
Followup to discussion in https://github.com/InsightSoftwareConsortium/ITKElastix/pull/173

### CODE REVIEW

**Review by dzenanz — APPROVED** [review]

**Review by thewtex — APPROVED** [review]

File: CMakeLists.txt [hunk 1]
```diff
@@ -35,10 +35,14 @@ endif()
 
 include(FetchContent)
 
+# Set up Elastix build parameters for populate step
+option(ELASTIX_BUILD_TESTING OFF)
+option(ELASTIX_BUILD_SHARED OFF)
```
Comment (N-Dekker) [inline CMakeLists.txt:40]: I like the proposed `ELASTIX_BUILD_TESTING` option, but I wonder, why do you propose to add `ELASTIX_BUILD_SHARED`? While I think the default (`OFF`) is fine, SuperElastix/elastix just does not support BUILD_SHARED_LIBS (`ON`), as far as I know... (See also https://github.com/SuperElastix/elastix/pull/283 "COMP: No longer add BUILD_SHARED_LIBS option").

Comment (tbirdso) [reply CMakeLists.txt:40]: Thanks for the catch @N-Dekker, I wrongly assumed Elastix supported shared libraries. Not a problem, will remove.

Comment (tbirdso) [reply CMakeLists.txt]: Updated.

**Review by N-Dekker — APPROVED** [review]
Thanks Tom! 👍 Thank you as well for addressing my comment on BUILD_SHARED_LIBS

**Comment by tbirdso** [comment]
Restarting some checks due to download timeout in macos CI

**Pull request merged**